align_anything/configs/train/text_audio_to_text/ppo.yaml

```diff
@@ -83,7 +83,7 @@ train_cfgs:
   freeze_vision_tower: True
   # Freeze the language model
   freeze_language_model: False
-# The data configurations
+# The logging configurations
```

Comment:
logging ?

Reply:
Yeah it is logging, fixed in https://github.com/PKU-Alignment/align-anything/pull/76/commits/ebfb956689624822788424b13b79de91b72e9a1f